frontend/src/queries/nodes/InsightViz/PropertyGroupFilters/PropertyGroupFilters.scss

```diff
@@ -0,0 +1,42 @@
+.property-group-filters {
```

Comment:
Might be worth refactoring to `.PropertyGroupFilters` in accordance to the [coding conventions](https://posthog.com/handbook/engineering/conventions/frontend-coding)?